.github/workflows/run-tests.yml

```diff
@@ -28,12 +33,16 @@ jobs:
             testbench: 8.*
             pest: 2.*
             collision: 7.*
+          - laravel: 11.*
+            testbench: 9.*
+            pest: 2.*
+            collision: 8.*
 
-    name: P${{ matrix.php }} - L${{ matrix.laravel }} - ${{ matrix.stability }} - ${{ matrix.os }}
+    name: PHP:${{ matrix.php }} / L:${{ matrix.laravel }}
```

Comment:
I don't think this name should be changed. It had everything we wanted already, including the stability and OS

Reply:
Reverted too